### PR TITLE
Services - fix cards - dev

### PR DESCRIPTION
**Pull request by Joanartdacosta**
- add cursor effect in all 'click' elements
- correct the style of service's card (fix alignment)

### CODE REVIEW

File: app/styles/global.css [hunk 1]
```diff
@@ -26,6 +26,9 @@ main {
 body {
   font-family: var(--font-paragraph);
 }
+.cursor {
```
Comment (doublethink13) [inline app/styles/global.css:29]: cursor-pointer

File: app/styles/global.css [hunk 2]
```diff
@@ -154,6 +167,12 @@ body {
 .width-18 {
   width: 18rem;
 }
+.max-width-15 {
```
Comment (doublethink13) [inline app/styles/global.css:170]: each of these attributes should be a separate class

File: app/styles/global.css (hunk 1 above)
Comment (Joanartdacosta) [reply app/styles/global.css]: done!

File: app/styles/global.css (hunk 2 above)
Comment (Joanartdacosta) [reply app/styles/global.css:170]: done!

**Pull request merged**